src/OrbitGl/App.cpp

```diff
@@ -1130,10 +1131,47 @@ orbit_base::Future<ErrorMessageOr<void>> OrbitApp::LoadModule(const std::string&
   const ModuleData* module_data = GetModuleByPath(module_path);
 
   if (module_data == nullptr) {
-    return {ErrorMessage{absl::StrFormat("Module \"%s\" was not found", module_path)}};
+    return ErrorMessage{absl::StrFormat("Module \"%s\" was not found", module_path)};
+  }
+
+  if (module_data->is_loaded()) {
```

Comment:
Is checking `is_loaded` actually the correct thing?
Or do I have to call `GetMutableTargetProcess()->IsModuleLoaded(module_data->file_path)`?

Reply:
Never mind. `is_loaded` is correct.